src/v2/view-builder/views/idp/BaseIdpAuthenticator.js

```diff
@@ -16,8 +17,11 @@ export const BaseIdpAuthenticatorView = BaseAuthenticatorView.extend({
     const messages = this.options.appState.get('messages') || {};
     // In case of failure, don't auto-redirect which will result in infinite redirects.
     // so catch the error and render to the user.
-    if (this.settings.get('features.skipIdpFactorVerificationBtn') && !Array.isArray(messages.value)) {
-      this.$('.o-form-button-bar').hide();
+    if (
+      this.settings.get('features.skipIdpFactorVerificationBtn') &&
+      !Array.isArray(messages.value) &&
+      this.model.get('formName') !== REMEDIATION_FORMS.REDIRECT_IDVERIFY
+    ) {  this.$('.o-form-button-bar').hide();
```

Comment:
nit: Place this statement on the next line instead of inline with the open brace. I'm surprised this didn't generate a lint error.